test/service/test_manifest.py

```diff
@@ -188,6 +188,10 @@ def test_manifest(self):
 
             ('file_content_type', 'application/pdf; dcp-type=data', 'application/gzip; dcp-type=data'),
 
+            ('drs_url',
```

Comment:
URI, not URL. See ticket. Be sure to understand the difference.

Reply:
I do recognize that you may have tried to maintain backwards compatibility. I just went ahead and renamed the columns in the BDBag.